sei-db/ss/pebbledb/db.go

```diff
@@ -570,7 +620,18 @@ func (db *Database) writeAsyncInBackground() {
 // NOTE: There is a rare case when a module's keys are skipped during pruning even though
 // it has been updated. This occurs when that module's keys are updated in between pruning runs, the node after is restarted.
 // This is not a large issue given the next time that module is updated, it will be properly pruned thereafter.
-func (db *Database) Prune(version int64) error {
+func (db *Database) Prune(version int64) (_err error) {
+	startTime := time.Now()
+	defer func() {
+		otelMetrics.pruneLatency.Record(
+			context.Background(),
+			time.Since(startTime).Seconds(),
+			metric.WithAttributes(
+				attribute.Int64("version", version),
```

Comment:
Same question about cardinality: how many unique versions are there in the lifetime of the program?